_data/errors.yml

```diff
@@ -0,0 +1,93 @@
+        - `http://idmanagement.gov/ns/assurance/ial/2` for Identity-Verification Permitted applications
+        Learn more about IAL values here: [https://developers.login.gov/oidc/#ial-values](https://developers.login.gov/oidc/#ial-values){:target="_blank"}
+    - title: The acr_values are not authorized
+      id: oidc-missing-acr-values
+      accordion-id: missing-acr
+      content: |
+        ###### Why it's happening
+        Your authentication request specified an Identity Verification (IdV) level of service, but your application is registered as "Authentication Only"
+        ###### What to do:
+        If your application is Authentication Only, check the value of your `acr_values` parameter and ensure that it does **not** include `http://idmanagement.gov/ns/assurance/ial/2`.
+        If you believe your application **SHOULD** be set to Identity Verification Permitted, please:
+        - Reach out to your account manager to make sure your application is authorized for IdV. If it isn't, they can work with you to determine the right service level for your application.
+        - If your application **is** authorized for IdV, go to the dashboard and update the Level of Service value to "Identity Verification Permitted"
+        - If this is a production instance, submit a [Zendesk request](https://zendesk.login.gov){:target="_blank"} to deploy that change to production.
+    - title: No acceptable acr_values found
+      id: oidc-no-acr-values
+      accordion-id: no-acr
+      content: |
+        ###### Why it's happening
+        No AAL levels are present in the acr_values parameter of the authentication request
+        ###### What to do:
+        - Check the `acr_values` parameter in your authentication request.
+        = Ensure that you are passing in the appropriate IAL value as well as any AAL values you might want for your application.
+        More information about the `acr_values` parameter can be found here: [https://developers.login.gov/oidc/#request-parameters](https://developers.login.gov/oidc/#request-parameters){:target="_blank"}
+    - title: No valid scope values found
+      id: oidc-no-scope
+      accordion-id: no-scope
+      content: |
+        ###### Why it's happening
+        No valid attributes were passed in the `scope` parameter of the authentication request.
+        ###### What to do:
+        - Check the `scope` parameter in your authentication request.
+        - Ensure you are passing in `openid` as well as at least one other attribute.
+        More information about the different types of attributes can be found here: [https://developers.login.gov/attributes/](https://developers.login.gov/attributes){:target="_blank"}
+    - title: No valid prompt values found
+      id: oidc-no-prompt
+      accordion-id: no-prompt
+      content: |
+        ###### Why it's happening
+        No prompt attribute was passed through the authentication request
```

Comment:
If still technically correct, I suggest changing this sentence to match the one above - "No prompt attribute was passed in the `prompt` parameter of the authentication request." If that is no longer correct please disregard.